packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js

```diff
@@ -77,18 +79,110 @@ describe('Component > TranscribedLines', function () {
   })
 
   describe('with all lines', function () {
+    let consensusComponents, createMarkSpy, returnRefs, showConsensusStub
+
     before(function () {
       sinon.spy(React, 'createRef')
-      wrapper = shallow(<TranscribedLines lines={consensusLines} task={task} />)
+      createMarkSpy = sinon.spy(TranscribedLines.prototype, 'createMark')
+      showConsensusStub = sinon.stub(TranscribedLines.prototype, 'showConsensus')
+      wrapper = mount(
+        <svg>
+          <TranscribedLines
+            lines={consensusLines}
+            task={task}
+            theme={zooTheme}
+          />
+        </svg>,
+        {
+          wrappingComponent: Provider,
+          wrappingComponentProps: {
+            classifierStore: {
+              workflows: {
+                active: {
+                  usesTranscriptionTask: false
+                }
+              }
+            }
+          }
+        }
+      )
+
+      returnRefs = React.createRef.returnValues
+      consensusComponents = wrapper.find(ConsensusLine)
     })
 
     after(function () {
+      task.setActiveMark(undefined)
       React.createRef.restore()
+      createMarkSpy.restore()
+      showConsensusStub.restore()
     })
 
     it('should call React createRef for each line', function () {
       expect(React.createRef.callCount).to.equal(consensusLines.length)
     })
+
+    it('should call ConsensusLine callback with expected ref on click', function () {
+      consensusComponents.forEach((consensusComponent, index) => {
+        const lineState = consensusComponent.find(TranscriptionLine).prop('state')
+        if (lineState === 'transcribed') {
+          expect(createMarkSpy.notCalled).to.be.true()
+          consensusComponent.simulate('click')
+          const [createMarkArgs] = createMarkSpy.args
+          expect(createMarkArgs[1]).to.equal(returnRefs[index])
```

Comment:
maybe set this to a variable for clarity? Perhaps `const expectedRefForMark = createMarkArgs[1]`?